docs/Backend_tutorials/features/easy_queries.md

```diff
@@ -0,0 +1,215 @@
+# How to create new EasyQuery
+
+This section provide basic information what needs to be implemented for new working easy query.
```

Comment:
I suggest put there some summary "what its `easy query`". For us its clear, but in public documentation should be some summary please...